winml/adapter/winml_adapter_apis.h

```diff
@@ -1,5 +1,5 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
+ // Licensed under the MIT License.
```

Comment:
This is a consequence of the recent change to add auto formatting to the winml folder (https://github.com/microsoft/onnxruntime/pull/16660). This is kind of buggy (every time you format it keeps indenting the comments instead of resetting them).

@justinchuby

Reply:
How was the file formatted? What is the clang-format version?

Reply:
I tried both vscode and lintrunner. Nothing happened 🤔

Reply:
My clang-format version is 16.0.1.  Applied with lintrunner version 0.10.7.

Reply:
Do you have any objection to reverting the formatting PR to unblock this?

Reply:
Yes. I think reverting and reapplying will be too costly considering the potential conflicts and CI time. I think it should be solvable by bumping the clang-format version. I can hop on a call with you for this

Reply:
The lint CI is currently green so the PR is not blocked. Looks like only the comments in the header files are incorrectly formatted, and there’s no code change in those files? Would it be possible to checkout main on those files? CI should still be green I think?

Reply:
Thanks for the parallel investigation on the lint issue.